readme.md

```diff
@@ -14,10 +14,16 @@ of software engineering:
 > Automatically maintain a repository of code that always passes all the tests.
 
 The application watches a repository for new pull requests. Once a pull request
-has been approved (through an LGTM comment left by a reviewer), it integrates
-the changes into master, and pushes those to a testing branch. When CI reports a
-successful build for this branch, master is forwarded to it. If the build fails,
-the commits never make it into master, keeping the build green at all times.
+has been approved (through an `@hoffbot merge` comment left by the PR author or
+reviewer), it integrates the changes into master, and pushes those to a testing
+branch. When CI reports a successful build for this branch, master is forwarded
+to it. If the build fails, the commits never make it into master, keeping the
+build green at all times.
+
+Merge trains: if a second PR receives a merge comment while a build is still
+happening, Hoff starts a speculative rebase on top of the previous build.  The
+same goes for a third PR.  Assuming all of the three builds pass, the third PR
+is merged after the time of a single build.
```

Comment:
Maybe we should shortly discuss the possible downsides of speculatively rebasing each pull request. That the choice was made for speculatively rebasing each individual PR which trades CI time for confidence that the individual PRs are buildable points in our history. As opposed to rollups that include multiple PRs in a single merge causing those rollups to act as the guaranteed buildable points in the history. I think that both the simplicity and additional confidence are worth the extra CI cost. The achieved integration speed might also be slightly higher because we don't have to split rollups into multiple separate builds upon failure.

Reply:
I have added some paraphrased version of your text in the discussion of one of the scenarios -- the one where the "conservative" choice is explained.